testflinger_agent/agent.py

```diff
@@ -128,6 +166,7 @@ def process_jobs(self):
             try:
                 job = TestflingerJob(job_data, self.client)
                 logger.info("Starting job %s", job.job_id)
+                self._post_json({"job_id": job.job_id})
```

Comment:
what about at the end of a test job when it's no longer running a job? we need to clear this back to ""